helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableNode.java

```diff
@@ -218,8 +206,25 @@ public int getAssignedReplicaCount() {
   /**
    * @return The current available capacity.
    */
-  public Map<String, Integer> getCurrentCapacity() {
-    return _currentCapacityMap;
+  public Map<String, Integer> getRemainingCapacity() {
+    return _remainingCapacity;
+  }
+
+  /**
+   * @return A map of <capacity category, capacity number> that describes the max capacity of the
+   *         node.
+   */
+  public Map<String, Integer> getMaxCapacity() {
+    return _maxAllowedCapacity;
+  }
+
+  public Map<String, Integer> getCapacityUsage() {
```

Comment:
I just noticed the usage of this method. Could you please move this interface implementation to where it is used? Not in this PR, please.

Reply:
Could you please confirm if you can avoid this unused method in this PR? You can add it when it's really used.